src/Compilers/CSharp/Portable/SymbolDisplay/ObjectDisplay.cs

```diff
@@ -262,7 +262,43 @@ public static string FormatLiteral(string value, ObjectDisplayOptions options)
                 throw new ArgumentNullException("value");
             }
 
-            return FormatString(value, options.IncludesOption(ObjectDisplayOptions.UseQuotes) ? '"' : '\0', escapeNonPrintable: true);
+            var quote = options.IncludesOption(ObjectDisplayOptions.UseQuotes) ? '"' : '\0';
```

Comment:
I believe the codegen is the same, so since quote is char, I'd just as soon see '\0', but I don't feel strongly.